### ISSUE TITLE
Scraper not working

### ISSUE THREAD

**Issue by Olifiers** [Aloshi/EmulationStation#800]
As of 24th of June 2019, none of the two scrapers are returning any matches, nor throwing any error messages when failing to scrap.

**Comment by ocket8888** [comment]
TheGamesDB.net now requires registered API keys and the legacy API has been shut down. I'm not sure this project is actually still maintained, but a good place to start would be modifying the request path in [GamesDBScraper.cpp](https://github.com/Aloshi/EmulationStation/blob/unstablees-app/src/scrapers/GamesDBScraper.cpp)

**Comment by ocket8888** [comment]
I'm attempting to fix this in #802 , but I think the maintainer of this project may never be around to merge it.

**Comment by DSMan97** [comment]
I propose a Solution for this I am goint to use https://api.igdb.com/ API to engace with the Scrapper on the Emulation Station, sorry for my English. 
I hope that Idea have contributors to develope

**Comment by ocket8888** [comment]
they have a 10k/month limit on the free version of their API, and probably have rules against disseminating a paid version to the public

**Comment by DSMan97** [comment]
So, what you propose?
We can Make It own Api for this but its so much Work.

**Comment by ocket8888** [comment]
For now I'm trying to work on giving people a way to configure their own scrapers, but with a default that uses TheGamesDB with my own public API key.

**Comment by DSMan97** [comment]
but legacy.thegamesdb.net/api/GetGame.php? is unavailable

**Comment by ocket8888** [comment]
sure, but api.thegamesdb.net *is* available

**Comment by DSMan97** [comment]
what?? and works equals, just changeing the String Url and compile Works?

**Comment by DSMan97** [comment]
Do you have a working version of emulations station with Scrapper Fix??

**Comment by ocket8888** [comment]
Sort of. I have this: #802 which works, assuming you have an API key to put in there. It doesn't have my API key yet, as I'm still testing. It also still includes the other scraper, which I haven't fixed yet.

**Comment by DSMan97** [comment]
In Retropie setup fork of EmulationStation is Fixed
see [here]( https://github.com/RetroPie/EmulationStation/blob/master/es-app/src/scrapers/GamesDBJSONScraperResources.cpp) i think, because today I tried and works (I don´t try months before because I thought that is not working) see my gif
![2019-07-31004125](https://user-images.githubusercontent.com/34092854/62170673-8f8fa700-b32c-11e9-9d9c-63052a111819.gif)


Did you fixed [ocket8888](https://github.com/ocket8888)?

**Comment by ocket8888** [comment]
Yes, Retropie seems to have solved this already. Dunno if they've already opened a PR, but it seems unlikely at this point that it'll get merged at any rate.

Mine works, but also is aiming in the long-term to be much more general, and to support any scraper database that matches a certain behavior pattern.

**Comment by TitanLife** [comment]
Is there a way to search and if multiple results choose between those that return? Kinda similar to hakchi?

**Comment by Vicj1** [comment]
What about Windows users? Any way to fix that?

**Comment by rfrayer420** [comment]
> What about Windows users? Any way to fix that?

I'm just gonna be honest ES is more worried about ARM builds. I understand why and in the big picture it is fair when you consider there isnt anywhere near as many programs for ARM. Doesnt make it suck any less. There are however ES forks that do work great because ES is after all opensource.
